pkg/apic/apiserver/models/catalog/v1/Category.go

```diff
@@ -27,8 +27,9 @@ var (
 )
 
 const (
-	CategoryResourceName        = "categories"
-	CategoryIconSubResourceName = "icon"
+	CategoryResourceName             = "categories"
+	Category_embeddedSubResourceName = "_embedded"
```

Comment:
This does not seem to be correct

Reply:
fixed up generate to ignore _embedded, and regen'd